src/ReservoirPriceOracle.sol

```diff
@@ -597,6 +611,7 @@ contract ReservoirPriceOracle is IPriceOracle, IReservoirPriceOracle, Owned(msg.
         assembly {
             sstore(lSlot, 0)
         }
+        // TODO: What about routs with length >4.
```

Comment:
Routes with length > 4 are not supported. 

`setRoute()` ensures that.

Reply:
Okay, we should add this comment to the code so the reader knows this

Reply:
okay added